src/main/java/core/basesyntax/MainApp.java

```diff
@@ -8,9 +8,26 @@
  * машина начала свою работу.
  * - Переопределить метод stopWork() и вывести в консоль строку с сообщением что та или иная
  * машина прекратила свою работу.
- * - В классе Main создать добавить по одному екземпляру каждого класса.
- * - Отправить все машини на работу и остановить работу всех машин.
+ * - В классе Main создать добавить по одному экземпляру каждого класса.
+ * - Отправить все машины на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
 
+        Bulldozer bulldozer =
```

Comment:
be attentive with [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)

Use abstract class to create Track, Bulldozer and Excavator instances.
Example:
Animal cat = new Cat();